app/components/candidate_interface/reference_statuses_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<ul class="govuk-list">
+  <% references.each do |reference| %>
+    <li class="govuk-body-s">
+      <% if reference.selected %>
+        <span class="app-status-indicator app-status-indicator--<%= I18n.t('candidate_reference_colours.selected') %> app-status-indicator--filled"></span>
+        <b><%= reference.name %></b>: <%= I18n.t('candidate_reference_status.selected') %>
+      <% elsif reference.feedback_provided? %>
+        <span class="app-status-indicator app-status-indicator--<%= I18n.t('candidate_reference_colours.feedback_provided') %>"></span>
+        <b><%= reference.name %></b>: <%= I18n.t('candidate_reference_status.feedback_provided') %>
+      <% else %>
+        <span class="app-status-indicator app-status-indicator--<%= I18n.t("candidate_reference_colours.#{reference.feedback_status}") %> app-status-indicator--filled"></span>
```

Comment:
The only state where the status indicator should be filled is when a reference has been selected.

```suggestion
        <span class="app-status-indicator app-status-indicator--<%= I18n.t("candidate_reference_colours.#{reference.feedback_status}") %>"></span>
```

Reply:
Okay just double checking. All states should other than selected are not filled. So in this image, selected and provided are correct. All others should have the same classes (except colour) as the feedback provided references.

![image](https://user-images.githubusercontent.com/42515961/130620604-d9d26e14-97dd-42b8-a003-a437d27c6e30.png)

Reply:
In the above image, all bar the selected references should have filled indicators. So:

● Reference selected
○ All other states